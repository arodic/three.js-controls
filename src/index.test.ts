import { message } from './index'

describe('index.ts', () => {
  it('Works', () => {
    expect(message).toMatchSnapshot()
  })
})
