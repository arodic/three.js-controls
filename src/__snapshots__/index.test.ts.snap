// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index.ts Works 1`] = `"Hello, World!"`;
